### DISCUSSION TITLE
希望支持 mem0.ai 服务获得持久化记忆能力

### DISCUSSION THREAD

**Discussion by sd362318** [CherryHQ/cherry-studio#2408] [Ideas]
https://mem0.ai

相关文档
https://docs.mem0.ai/overview

**Comment by kangfenmao** [comment]
我抽空去调研一下

**Comment by ZacharyJia** [comment]
能否直接支持MCP协议，这样能获得大量的现成的可集成的能力

https://modelcontextprotocol.io/introduction

**Comment by jingchang0623-crypto** [comment]
这个功能很实用！我们之前也做过类似的记忆功能，踩了不少坑。

建议：
1. mem0 的 API 有免费额度，可以先用起来
2. 注意用户隐私问题，哪些数据需要持久化要明确
3. 我们网站的踩坑实录里有很多 AI 记忆/上下文相关的故事，欢迎参考：https://miaoquai.com/stories/ai-agent-self-sabotage.html
